cpp/ps0000/cpp_ps0000.cpp

```diff
@@ -1,7 +1,8 @@
-#include <iostream>
+#include <iostream> // 2/2
 using namespace std;
-int Main()
+int Main() // 0/1
 {
-  cout << "Hello World"<<"\n";
-  return 0;
-}
\ No newline at end of file
+  cout << "Hello, World\n"; // 3/4
+  return 0; // 1/1
+}
+// // great work 😎 8/10 ⭐

```

Comment:
- [x] most of code errors and bugs are fixed
- [x] file is in the right path
- [x] +1 mark for your first commit